.github/workflows/ci.yaml

```diff
@@ -13,11 +13,11 @@ on:
 
 env:
   # Qt: 5.15.0
-  # libtorrent: RC_1_2 HEAD, 1.2.10
+  # libtorrent: RC_1_2 HEAD, 1.2.11
   VCPKG_COMMIT: 32eccc18191fbb57b159784a1724d2d00613ae82
   VCPKG_DEST_MACOS: /Users/runner/qbt_tools/vcpkg
   VCPKG_DEST_WIN: C:\qbt_tools\vcpkg
-  UBUNTU_LIBTORRENT_VERSION: libtorrent-1.2.10
+  UBUNTU_LIBTORRENT_VERSION: libtorrent-1.2.11
```

Comment:
libtorrent uses inconsistent version naming so github actions CI still fails.
This should be ` v1.2.11` instead, please give it one last try.

Reply:
>libtorrent uses inconsistent version naming so github actions CI still fails.
This should be v1.2.11 instead, please give it one last try.

Done.

Reply:
Mac builds still fail. Should I change something else to fix them?

Reply:
>Mac builds still fail. Should I change something else to fix them?

The mac & windows builds uses vcpkg and vcpkg hasn't update libtorrent to 1.2.11 yet. There is nothing we can do here.

Reply:
Should we wait for vcpkg and AppVeyor support for libtorrent-1.2.11?

Reply:
>Should we wait for vcpkg and AppVeyor support for libtorrent-1.2.11?

In general yes, but I also don't mind if you merge it now.

Reply:
I don't know how fast vcpkg follows the release of new versions... maybe @FranciscoPombal can clear it up.
But what really bothers me is that AppVeyor is updated manually by @sledgehammer999. I don't expect him to be able to do it quickly. And since we decided to follow the libtorrent updates closer, this may be a problem. Maybe we can give AppVeyor up altogether?

Reply:
>I don't know how fast vcpkg follows the release of new versions... 

That depends on contributors. We might have to participate if we want it updated fast.
https://github.com/microsoft/vcpkg/tree/master/ports/libtorrent

And it sometimes may not be suitable for us: https://github.com/microsoft/vcpkg/pull/14293

Reply:
I'm still skeptical about vcpkg, especially the prospect of having it build official releases. On the one hand, it certainly increases convenience (especially useful for automation), but on the other hand, it binds our hands. Well, either we have to maintain our own ports for used libraries.

Reply:
>Well, either we have to maintain our own ports for used libraries.

Also note that libtorrent version bump is mostly done by one person: https://github.com/microsoft/vcpkg/commits/master/ports/libtorrent
It shows that there are not many users out there that are interested to keep the port up-to-date.

Reply:
@Chocobo1 @glassez 

I can submit a PR to update the libtorrent port upstream either today or tomorrow. In the meantime, we should show support for this comment: https://github.com/microsoft/vcpkg/pull/14293#issuecomment-731724895.

Reply:
If we are serious about having vcpkg in our build toolset, we should have more influence on libtorrent port maintenance.